### PR TITLE
Remove unused localization strings

### PR DESCRIPTION
**Pull request by lostmypillow**
**Changes**

**Issues**

### CODE REVIEW

File: src/strings/en-us.json [hunk 1]
```diff
@@ -339,10 +333,6 @@
     "EnablePhotosHelp": "Photos will be detected and displayed alongside other media files.",
     "EnableStreamLooping": "Auto-loop live streams",
     "EnableStreamLoopingHelp": "Enable this if live streams only contain a few seconds of data and need to be continuously requested. Enabling this when not needed may cause problems.",
-    "EnableThemeSongs": "Enable theme songs",
-    "EnableThemeSongsHelp": "If enabled, theme songs will be played in the background while browsing the library.",
-    "EnableThemeVideos": "Enable theme videos",
-    "EnableThemeVideosHelp": "If enabled, theme videos will be played in the background while browsing the library.",
```
Comment (cvium) [inline src/strings/en-us.json]: All four of these are still referenced in the code

File: src/strings/en-us.json [hunk 2]
```diff
@@ -1501,16 +1478,11 @@
     "MessageServerConfigurationUpdated": "Server configuration has been updated",
     "MessageSettingsSaved": "Settings saved.",
     "MessageThankYouForConnectSignUp": "Thank you for signing up for Jellyfin Connect. An email will be sent to your address with instructions on how to confirm your new account. Please confirm the account and then return here to sign in.",
-    "MessageThankYouForConnectSignUpNoValidation": "Thank you for signing up for Jellyfin Connect! You will now be asked to login with your Jellyfin Connect information.",
+ "MessageThankYouForConnectSignUpNoValidation": "Thank you for signing up for Jellyfin Connect! You will now be asked to login with your Jellyfin Connect information.",
```
Comment (cvium) [inline src/strings/en-us.json]: This one should've been removed, right?

File: src/strings/en-us.json (hunk 1 above)
Comment (lostmypillow) [reply src/strings/en-us.json]: I thought it wasn't working, what should I do anyway?

File: src/strings/en-us.json (hunk 2 above)
Comment (lostmypillow) [reply src/strings/en-us.json]: It wasn't, I added it back

**Comment by JustAMan** [comment]
@lostmypillow why you have closed the PR, it wasn't merged yet?

**Pull request closed without merging**